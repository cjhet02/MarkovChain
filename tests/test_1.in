test1.txt
0
0
