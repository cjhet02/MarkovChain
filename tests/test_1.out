Enter cleaned data file name: due - 1: to - 100% 
to - 1: the - 100% 
the - 2: nature - 50% project - 50% 
nature - 1: of - 100% 
of - 1: the - 100% 
project - 1: , - 100% 
, - 1: pre-packaged - 100% 
pre-packaged - 1: tests - 100% 
tests - 1: only - 100% 
only - 1: useful - 100% 
useful - 1: check - 100% 
check - 1: adjacency - 100% 
adjacency - 1: lists - 100% 
lists - 1: . - 100% 
. - 0: 

Enter desired sentence length: 
Enter desired number of sentences: 